src/sentry/utils/snuba.py

```diff
@@ -29,6 +29,18 @@
 MAX_HASHES = 5000
 
 SENTRY_SNUBA_MAP = {
+    # general
+    'event_id': 'event_id',
+    'project_id': 'project_id',
+    'message': 'message',
+    'issue': 'issue',
+    'group_id': 'group_id',
+    'timestamp': 'timestamp',
+    'time': 'time',
+    'deleted': 'deleted',
+    'retention_days': 'retention_days',
+    'type': 'type',
+    'version': 'version',
```

Comment:
i added a couple of these here: https://github.com/getsentry/sentry/pull/10503, which i just merged

also, is issue actually a column? i don't have it locally, but could also be that my snuba is outdated

Reply:
`issue` is a computed column, not actually in the clickhouse table. There is a little bit about it in the [readme](https://github.com/getsentry/snuba/blob/master/README.md)